conanfile.py

```diff
@@ -10,10 +10,10 @@ class VTKConan(ConanFile):
     SHORT_VERSION = short_version
     generators = "cmake"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "qt": [True, False]}
-    default_options = "shared=False", "qt=False"
+    options = {"shared": [True, False], "qt": [True, False],"cygwin_msvc": [True, False],"linux_use_sudo": [True, False]}
```

Comment:
Please separate new options by a whitespace, or better by splitting this long line into multiple lines.